components/brave_wallet_ui/page/wallet_page.html

```diff
@@ -11,6 +11,11 @@
   <script src="/brave_wallet_page.bundle.js"></script>
   <script type="module" src="chrome://brave-resources/br_elements/br_toolbar/br_toolbar.js"></script>
   <style>
```

Comment:
at some point (in this PR if it's easy) - could we move all these inline styles to their own style file so that we can remove unsafe-inline `style-src` from the CSP? cc @petemill

Reply:
> see comments above

@diracdeltas I have made requested changes.

Reply:
sorry not sure what this is referring to; i still see inline styles in the `<style>` element in this file. it's not a big deal but given that eventually styles should ideally be moved into their own files, makes sense to do that now.